### ISSUE TITLE
Resolving lower compat bounds

### ISSUE THREAD

**Issue by adrhill** [StefanKarpinski/Resolver.jl#5]
I stumbled upon Resolver in your recent [Discourse comment](https://discourse.julialang.org/t/what-is-the-pkg-jl-package-dependency-resolver-solver-algorithm/77821/5), in which you mention:

> Unlike the current algorithm (or SAT solvers), which only finds one solution, it finds all Pareto-optimal solutions.

This reminded me of recent discussions on testing of lower-bound compat entries in https://github.com/JuliaLang/Pkg.jl/pull/3802, where Pareto optimality also popped up.

Resolving lower compat bounds currently requires manual bisection. Could this package help here?

**Comment by StefanKarpinski** [comment]
Yes, this does support that. See the [`--min` option](https://github.com/StefanKarpinski/Resolver.jl/blob/4f371082e965372fd679fa9a2562f8a7ae34081d/bin/resolve.jl#L33) for the `bin/resolve.jl` script.
